pkg/ingester/ingester_test.go

```diff
@@ -100,6 +100,7 @@ func TestIngester_Push(t *testing.T) {
 		maxExemplars              int
 		maxMetadataPerUser        int
 		maxMetadataPerMetric      int
+		acceptNativeHistograms    bool
```

Comment:
Note to reviewers: we can add more tests for native histograms here when ingester returns them, but currently you cannot consistently encode histogram related testcases as counters/statistics do not match what the query returns!